### ISSUE TITLE
Make SelectOne Player Advanced Modifiers selectable without hitting select

### ISSUE THREAD

**Issue by aeubanks** [dguzek/Simply-Love-SM5#419]
On the main modifiers menu, when choosing a modifier that only has one possible value, you only have to hit left/right to choose the value. But on the advanced modifiers menu, you have to hit left/right, then hit select to choose. The main modifier menu version seems nicer, and this would make things more consistent.

**Comment by yyr57695** [comment]
I disagree that this suggested change would make things more consistent overall.

The Main Modifiers menu does not contain any options allowing multiple selections; literally everything in Main Modifiers can only be set to one value.  This is why you only have to hit left/right.

The Advanced and Uncommon Modifiers menus, by comparison, each have several options allowing multiple selections, especially Uncommon Modifiers where almost every option does.  Because of this, you have to press the select button to toggle each selection on/off.  It is true that this means different behavior from Main Modifiers.  However, it is consistent across the entirety of those two menus.  Once you start using those two menus, and especially once you begin using the options with multiple selections, the behavior starts to make sense.

If the behavior were to be changed to what you suggest, then some options on Advanced Modifiers would have different behavior from others.  In other words, the behavior of the Advanced Modifiers menu would no longer be consistent with itself.  For newer players, it is not always obvious which options have multiple selections, so this may wind up being more confusing than the previous setup.

A "happy medium" solution would either be to merge the single-selection Advanced Modifiers into Main Modifiers, or to split Advanced Modifiers into 2 separate screens, one for the single-selection options and one for the multi-selection options.  But I don't personally feel that either of these are good solutions, because IMO, Main Modifiers should be kept to one screen, and 3 screens of options ought to be enough for anybody.

Ultimately, in my opinion, the current behavior is OK.  But it'd be great if some of the more frequent players could weigh in.
